R/fitting-and-postprocessing.R

```diff
@@ -16,12 +15,10 @@ sample_model <- function(model, data, scenario = data.table::data.table(id = 1),
   fit <- safe_fit_model(model, data, ...)
 
   if (!is.null(fit$error)) {
-    out <- out |>
-      DT(, error := list(fit$error[[1]]))
+    out[, error := list(fit$error[[1]])]
```

Comment:
an annoying side effect of linting. Need to add error as a global var in utils to get rid of this flag.

Reply:
or we can update the linter to ignore global var checks (but then Rmd check will still flag this anyway)

Reply:
alternatively the env features we use elsewhere may help here?